### ISSUE TITLE
Ability to apply CSS rules to the rendered element(s)

### ISSUE THREAD

**Issue by frank-rollpin** [dart-lang/markdown#238]
It would be great to be able to apply css rules to the rendered elements. In particular, this feature would be quite handy for adding image thumbnails:

https://gist.github.com/uupaa/f77d2bcf4dc7a294d109
```
![test image size](/img/post-bg-2015.jpg){:class="img-responsive"}
![test image size](/img/post-bg-2015.jpg){:height="50%" width="50%"}
![test image size](/img/post-bg-2015.jpg){:height="700px" width="400px"}
```

**Comment by srawlins** [comment]
If this is a specified syntax on some standard syntax specification, like GFM or Pandoc or CommonMark, we could implement this. I've not seen this syntax before though.

**Comment by frank-rollpin** [comment]
It appears that inline attributes are part of the kramdown (used on github) specification. It would indeed be great to have this implemented.

https://kramdown.gettalong.org/quickref.html#inline-attributes

**Comment by srawlins** [comment]
Hmm I don't see it in the spec. https://github.github.com/gfm/#images

It also doesn't appear to work in this comment box:

```markdown
![a](b){:class="img-responsive"}
```

![a](b){:class="img-responsive"}

**Comment by frank-rollpin** [comment]
You are right, it does not look like GitHub supports it anymore (although it appears that it did, based on the comments [here](https://gist.github.com/uupaa/f77d2bcf4dc7a294d109)). Anyway, this seems to be a part of the kramdown standard, and is quite a useful feature that generalizes well - not only it solves the popular issue with the image size, but also provides the only way to style resulting html. I think it will be a great addition to the library.
